mapfishapp/README.md

```diff
@@ -56,10 +56,21 @@ It is also possible to POST a JSON string to the home controller, for instance :
             "layername": "ign:ign_bdtopo_region",
             "owstype": "WMS",
             "owsurl": "http://ids.pigma.org/geoserver/ign/wms"
-        }]
+        }],
+        "search":{
+    		"owsurl":"http://ids.pigma.org/geoserver/ows",
+    		"cql_filter": "id_dept = 48",
+    		"featurens":"http://ids.pigma.org/geoserver/ign",
+    		"typename":"ign_bdtopo_departement"
```

Comment:
I don't think it's good to set prefix in typeName, as there is a field for setting this (featurePrefix).

When I developed an addon for search accross different zones/referentials, I have to set the featurePrefix param, because if I use only the featureType, in the raquest I have "feature:referentiels:communes", because it append featureNS with default prefix value.

It happen with the OpenLayers.Protocol.WFS_1_1_0 read method, but I haven't tested with 2_0_0

But effectively the featureNS can be removed if we have a featurePrefix provided